src/ui/Counter.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faMinusCircle, faPlusCircle } from '@fortawesome/free-solid-svg-icons'
+
+const CounterLabelStyle = styled.div`
+  width: 72px;
+  height: 24px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  border: 1px solid #c4c4c4;
+  border-radius: 6px;
+`
+
+const CounterStyle = styled.input`
+  margin: 0;
+  width: 20px;
+  border: none;
+  color: black;
+  font-weight: 500;
+  font-size: 0.75rem;
+  text-align: center;
+  font-family: 'Merriweather Sans', -apple-system, BlinkMacSystemFont,
```

Comment:
Is this reqlly necessary? Fonts are set in the GlobalStyle

Reply:
Well noted